mantidproject/mantid
# Correct angles for final banks in IN6 IDF

A number of angles for the detector boxes were wrong in the IN6 IDF for banks > 102 degrees. This fixes that issue.

This PR also groups the detectors into top, middle and bottom banks.

**To test:**

Load up an IN6 data file (for example `/ExternalData/Testing/Data/UnitTest/ILL/IN6/164192.nxs`). Check that for higher bank numbers the combinations of theta and phi are unique (a given theta angle is used for a maximum of 3 detectors, each at a differnt phi angle). For example detectors 299 - 301 and 308 - 310 should now be different.

Check that in the instrument hierarchy the detector group contains `top_bank`, `middle_bank` and `lower_bank`.

Please also merge https://github.com/mantidproject/mantidgeometry/pull/87 along with this.

Fixes #19079 .

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
